src/pages/evm/nfts/NftInventoryPage.vue

```diff
@@ -447,7 +447,7 @@ onUnmounted(() => {
                                 @click="goToDetailPage(props.row)"
                                 @keydown.space.enter.prevent="goToDetailPage(props.row)"
                             >
-                                <template v-if="props.row.image">
+                                <template v-if="props.row.image || props.row.isVideo">
```

Comment:
fix the issue where the hover preview for NFT tile icons would not show for video NFTs